packages/syft/src/syft/lib/torch/module.py

```diff
@@ -420,16 +423,33 @@ def debug_sum_layers(self) -> None:
                         info(f"  Layer {n} sum({k}): {s}")
 
 
+def module_in_ast(module_type: type) -> bool:
```

Comment:
Can you move this to `syft/lib/ast/util.py`?

Reply:
(outdated - came from anotheor PR)